metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -223,13 +232,42 @@ class WorksheetProvider(
     )
   }
 
+  private def fetchDependencySources(
+      dependencies: Seq[Dependency]
+  ): List[Path] = {
+    Fetch
+      .create()
+      .withDependencies(dependencies: _*)
+      .addClassifiers("sources")
+      .fetchResult()
+      .getFiles()
+      .map(_.toPath())
+      .asScala
+      .toList
+  }
+
   private def evaluateWorksheet(
       path: AbsolutePath,
       token: CancelToken
   ): EvaluatedWorksheet = {
     val mdoc = getMdoc(path)
     val input = path.toInputFromBuffers(buffers)
-    val worksheet = mdoc.evaluateWorksheet(input.path, input.value)
+    val relativePath = path.toRelative(workspace)
```

Comment:
that was this incompatibility with newest mdoc? :laughing:

Reply:
That's the change for worksheet to be able to include dependencies.